docs/source/developer/css.rst

```diff
@@ -191,3 +194,40 @@ only the desired children.
 
 When in doubt, there is little harm done in parents adding selectors to
 children.
+
+Commonly useful CSS selectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following common CSS class names are treated as public.
+
+**CSS classes that target widgets and their children**
+-  ``jp-Activity``: applied to elements in the main work area
+-  ``jp-Cell``: applied to cells
+-  ``jp-CodeCell``: applied to code cells
+-  ``jp-CodeConsole``: applied to consoles
+-  ``jp-CodeConsole-content``: applied to content panels in consoles
+-  ``jp-CodeConsole-promptCell``: applied to active prompt cells in consoles
+-  ``jp-DirListing-content``: applied to contents of file browser directory listings
+-  ``jp-DirListing-item``: applied to items in file browser directory listings
+-  ``jp-FileEditor``: applied to file editors
+-  ``jp-ImageViewer``: applied to image viewers
+-  ``jp-InputArea-editor``: applied to cell input area editors
+-  ``jp-Notebook``: applied to notebooks
+-  ``jp-SettingEditor``: applied to setting editors
+-  ``jp-SideBar``: applied to sidebars
+-  ``jp-Terminal``: applied to terminals
+**CSS classes that describe the state of a widget**
+-  ``jp-mod-current``: applied to elements on the current document only
+-  ``jp-mod-completer-enabled``: applied to ediors that can host a completer
+-  ``jp-mod-commandMode``: applied to a notebook in command mode
+-  ``jp-mod-editMode``: applied to a notebook in edit mode
+-  ``jp-mod-has-primary-selection``: applied to editors that have a primary selection
+-  ``jp-mod-in-leading-whitespace``: applied to editors that have a selection within the beginning whitespace of a line
+-  ``jp-mod-tooltip``: applied to the body when a tooltip exists on the page
+**CSS classes that describe elements**
+-  ``[data-jp-code-runner]``: applied to widgets that can run code
+-  ``[data-jp-interaction-mode=”terminal”]``: applied when a terminal is in use
+-  ``[data-jp-interaction-mode=”notebook”]``: applied when a notebook is in use
```

Comment:
This is applied when a code console is in notebook ("Shift Enter to Execute") mode.